proto/panacea/oracle/v2alpha2/oracle.proto

```diff
@@ -91,4 +91,27 @@ message VotingPeriod {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"voting_end_time\""
   ];
+}
+
+// OracleRegistrationVote defines the vote for oracle registration
+message OracleRegistrationVote {
+  string unique_id = 1;
+  string voter_address = 2;
+  string voting_target_address = 3;
+  VoteOption vote_option = 4;
+  bytes encrypted_oracle_priv_key = 5;
+}
+
+// VoteOption enumerates the vote option.
+enum VoteOption {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // VOTE_OPTION_UNSPECIFIED
+  VOTE_OPTION_UNSPECIFIED = 0;
+
+  // VOTE_OPTION_VALID
+  VOTE_OPTION_VALID = 1;
+
+  // VOTE_OPTION_INVALID
+  VOTE_OPTION_INVALID = 2;
```

Comment:
I would go with `yes` and `no` which are more general than `valid` and `invalid` because these vote options can be used in the future for certain vote types that don't require any 'validation'. But, it's not a big deal now.

Reply:
OK. `yes`, `no` consider the change later